---
string current_place
string current_status
int64 robot_id
